test/test/sklearndf/test_sklearn_coverage.py

```diff
@@ -57,11 +59,20 @@
     *GENERAL_COVERAGE_EXCLUSIONS,
     # class "Imputer" was deprecated in 0.20 and removed in 0.22
     "Imputer",
+    "AgglomerationTransform",  # TODO should be?
```

Comment:
I agree this should not be wrapped, given it is a mix-in class. Could you please add a comment in the line above with a short explanation for the exclusion? Similar to how we explain why we exclude `Imputer`.

Reply:
Sure! I added explanation for that.